src/app/modules/shared/components/breadcrumb/breadcrumb.service.ts

```diff
@@ -0,0 +1,54 @@
+import { Injectable, EventEmitter } from '@angular/core'
+import { Router, ActivatedRouteSnapshot, Event, NavigationEnd, UrlSegment } from '@angular/router'
+import { Breadcrumb } from './breadcrumb.model'
+
+@Injectable()
+export class BreadcrumbService {
+  breadcrumbChanged = new EventEmitter<Breadcrumb[]>(false);
```

Comment:
EventEmitter is a pattern from declarative, normal javascript. In this case, we are trying to leverage rxjs to use functional reactive programming. The way to do so is using an rxjs `Subject` https://rxjs-dev.firebaseapp.com/guide/subject. Also, the variable naming pattern for observables (subjects are observables) is something like `breadcrumbChange$`